app/views/microcosms/_form.html.erb

```diff
@@ -0,0 +1,70 @@
+<%# if microcosm.errors.any? %>
+<!--    <div id="error_explanation">-->
+<!--      <h2><%#= pluralize(microcosm.errors.count, "error") %> prohibited this microcosm from being saved:</h2>-->
+<!--      <ul>-->
+          <%# microcosm.errors.full_messages.each do |message| %>
+<!--          <li><%#= message %></li>-->
+          <%# end %>
+<!--      </ul>-->
+<!--    </div>-->
+<%# end %>
+
+<%= form_for @microcosm do |form| %>
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name, id: :microcosm_name %>
+  </div>
+
+  <div class="field">
+    <%= form.label :location %>
+    <%= form.text_field :location, id: :microcosm_location %>
+  </div>
+
+  <div class="field">
+    <%= form.label :lat %>
+    <%= form.text_field :lat, id: :microcosm_lat %> (decimal)
+  </div>
+
+  <div class="field">
+    <%= form.label :lon %>
+    <%= form.text_field :lon, id: :microcosm_lon %> (decimal)
+  </div>
+
+  <p>
+    To get a bounding box use Geofabrik's <a href="http://tools.geofabrik.de/calc/#type=geofabrik_standard&bbox=-120.848658,31.674368,-79.375023,49.413863&tab=1&proj=EPSG:4326&places=2">Tile Calculator</a> tool.
+  </p>
```

Comment:
I don't know what the solution is offhand but I don't think we want to be referring people to a third party service like this.

Reply:
Got it.

Reply:
You might use https://github.com/kajic/leaflet-locationfilter to allow users select the bounding box. The Leaflet plugin is used at https://tyrasd.github.io/osm-qa-feeds/ (you have to search for a location first).

Reply:
We already have leaflet-location filter installed - it is used for selecting the area to export.

Reply:
For now we can just just grab numbers from "Export" button at top of osm.org.